### PR TITLE
Prepare docs for sponsoring

### PR DESCRIPTION
**Pull request by oe**
I'll update this with the correct post date before it gets merged. Please review the post, the sponsor logo placement has been confirmed with Forestry.

### CODE REVIEW

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: 'Jekyll and Sponsoring'
+date: 2018-06-26 16:00:00 +0200
+author: oe
+categories: [community]
+---
+
+_(TL;DR: We're open for sponsorships over our [OpenCollective page](https://opencollective.com/jekyll))_
```
Comment (ashmaroli) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: `for sponsorships at our..` ?

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+interface, and then automatically commits it back to your GitHub repository. We're excited to have
+them on board on a new, exciting step of our journey.
+
+Will anything change for Jekyll users? The answer is no - this step does not impact the Jekyll software
+in any aspect. In fact, you might see positive changes, such as more features and better
+performance. Surprisingly, that's what happens when you properly fund people for their work!
+
+If you have been a long time user for Jekyll and would like to give something back to the project,
+you can consider a small monthly donation to our [OpenCollective page](http://opencollective.com/jekyll).
+If your company heavily relies on Jekyll, consider sponsoring us! Contact me at
```
Comment (ashmaroli) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: `please consider sponsoring us` ?

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+`olivia at fastmail dot com` and we'll figure something out together.
```
Comment (ashmaroli) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: why not simply `olivia@fastmail.com`?

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown (hunk 2 above)
Comment (oe) [reply docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: well we don't want to sound like we're begging for it :wink:

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown (hunk 3 above)
Comment (oe) [reply docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: our site gets a bunch of traffic which means that there is usually some bots that scour for email addresses, and who knows what they do with that

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown (hunk 2 above)
Comment (ashmaroli) [reply docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: `do consider sponsoring us!` then.. :stuck_out_tongue:

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown (hunk 3 above)
Comment (ashmaroli) [reply docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: fair point.

File: docs/_data/sponsors.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+- name: Forestry.io
+  image: /img/forestry-logo.png
+  url: https://forestry.io
```
Comment (ashmaroli) [inline docs/_data/sponsors.yml]: personal nit: Can we have a blank newline here please?

Comment (oe) [reply docs/_data/sponsors.yml]: oh of course

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
+---
+title: 'Jekyll and Sponsoring'
```
Comment (DirtyF) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: Switch to active voice ?

- Start sponsoring Jekyll’s development

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+a certain plugin or feature. Today, we're excited to announce that we will finally
+be able to fund our contributors! __We are opening an OpenCollective to receive
+individual and corporate sponsorships__. This is not unheard of, [Hugo](http://gohugo.io)
+is also funded by sponsorships, as are many other similar projects.
+
+OpenCollective is a service that makes it easy for open source projects to receive funding
+from individuals and companies alike. It's specifically designed for open source and
+many other projects already use it for funding.
+
+With that, we would like to announce our very first sponsor: [__Forestry.io__](https://forestry.io)! Forestry is
```
Comment (DirtyF) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: Let's add their logo here to create a visual break and give them visibility

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+software that, ironically, is being used by almost every company that offers
+digital services. It's a problem that has gotten more attention in recent years, with
+the open source community becoming more diverse and more and more companies actively
+investing in providing monetary support for open source developers.
+
+Jekyll has always been a product of volunteers. Rarely has someone been paid to implement
+a certain plugin or feature. Today, we're excited to announce that we will finally
+be able to fund our contributors! __We are opening an OpenCollective to receive
+individual and corporate sponsorships__. This is not unheard of, [Hugo](http://gohugo.io)
+is also funded by sponsorships, as are many other similar projects.
```
Comment (DirtyF) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: link to one or more opencollective projects here

File: docs/_posts/2018-06-26-jekyll-sponsoring.markdown [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+software is effectively the result of hundreds and thousands of hours of free labor provided
+by individuals who are passionate enough to work outside of their day job to create
+software that, ironically, is being used by almost every company that offers
+digital services. It's a problem that has gotten more attention in recent years, with
+the open source community becoming more diverse and more and more companies actively
+investing in providing monetary support for open source developers.
+
+Jekyll has always been a product of volunteers. Rarely has someone been paid to implement
+a certain plugin or feature. Today, we're excited to announce that we will finally
+be able to fund our contributors! __We are opening an OpenCollective to receive
```
Comment (DirtyF) [inline docs/_posts/2018-06-26-jekyll-sponsoring.markdown]: Add Jekyll's open collective screenshot, once we have received our first donation ?

**Pull request merged**